src/components/Layout/AppLayout.test.jsx

```diff
@@ -0,0 +1,86 @@
+import { composeStories } from "@storybook/testing-react";
+import { render } from "@testing-library/react";
+import { axe } from "jest-axe";
+import { vi } from "vitest";
+
+import * as AppLayoutStories from "./AppLayout.stories";
+
+// the following defineProperty call is needed to woraround an issue with the test library.
```

Comment:
can you add a link that describes what the issues is? also, spelling issue
```suggestion
// the following defineProperty call is needed to workaround an issue with the test library.
```

Reply:
done